docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -37,18 +36,23 @@ journey("My Journey", ({ page, params }) => {
 If you try to run the example above you'll notice an error because we haven't specified a value for the `url` parameter.
 Parameter values can be declared by any of the following methods:
 
-* Declaring a default value for the parameter in a configuration file.
-* Passing the `--params` CLI argument. 
-* Specifying the parameter in the {heartbeat} YAML config using the `params` option.
+* In the _Global parameters_ tab of the <<synthetics-settings-global-parameters,Synthetics Settings page in {kib}>>.
+* Declaring a default value for the parameter in a <<synthetics-dynamic-configs,configuration file>>.
+* Passing the `--params` <<synthetics-cli-params,CLI argument>>. 
 
-The values in the configuration file are read first, then merged with either the CLI argument, or the {heartbeat}
-option, with the CLI and {heartbeat} options taking precedence over the configuration file.
+// To do: Need to verify
+The values in the configuration file are read in the following order:
 
-These options are discussed in detail in the sections below.
+. *Global parameters*: The Global parameters set in {kib} are read first.
+. *Configuration file*: Then the Global parameters are merged with any parameters defined in a configuration file.
+  If a parameter is defined in both {kib} and a configuration file,
+  the value in the configuration file will be used.
+. *CLI*: Then the parameters defined in the configuration are merged with any parameters passed to the CLI `--params` argument.
+  If a parameter is defined in {kib} or a configuration file *and* using the CLI argument,
```

Comment:
Ah good point. Do you think it's worth saying that or is it implied? 

>3. **CLI**: Then the parameters defined in the configuration are merged with any parameters passed to the CLI `--params` argument. If a parameter is defined in a configuration file *and* using the CLI argument, the value defined using the CLI will be used. When running a script using the CLI, Global parameters defined in {kib} have no impact on the test because it won't have access to the {kib}-defined global parameters.

Reply:
Yeah I think this looks good - it makes it very clear (as it may well not be obvious that Kibana params won't be available to the CLI)